validator/client/src/main/java/tech/pegasys/teku/validator/client/loader/graffiti/Bytes32Converter.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.validator.client.loader.graffiti;
+
+import java.nio.charset.StandardCharsets;
+import org.apache.tuweni.bytes.Bytes32;
+
+public class Bytes32Converter {
+  public static Bytes32 toBytes32(final byte[] input) {
+    if (input.length > 32) {
+      throw new IllegalArgumentException(
+          "'"
+              + new String(input, StandardCharsets.UTF_8)
+              + "' converts to "
+              + input.length
+              + " bytes. Input must be 32 bytes or less.");
+    }
+
+    // TODO note to self and code reviewer. Is this implementation better or worse than
+    //     return Bytes32.rightPad(Bytes32.wrap("myString".getBytes(UTF_*)));
```

Comment:
Looks like `Bytes32.wrap` expects the byte array to have exactly 32 bytes, so I don't think this will work.   With the current implementation we're allocating a single byte array which should be faster than creating a smaller `Bytes` object and then resizing it.